submit/submit.go

```diff
@@ -207,15 +208,15 @@ func Main() {
 			n = 2
 		}
 		if len(ebs) == 4 {
-			ebsCmd[0] = fmt.Sprintf("export vid=$(batchit ebsmount -n %d -m %s -s %s -v %s -t %s)", n, ebs[0], ebs[1], ebs[2], ebs[3])
+			ebsCmd[1] = fmt.Sprintf("export vid=$(batchit ebsmount -n %d -m %s -s %s -v %s -t %s)", n, ebs[0], ebs[1], ebs[2], ebs[3])
```

Comment:
this should be now ebs 1 through 4. but better yet. don't prepend the function to the ebsCmd array, just add the new string to the arguments list in the sprintF before ebsCmd[0].